docs/architecture/baking.md

```diff
@@ -27,25 +27,62 @@ It packages pending operations into a block, signs it, and distributes it to the
-The more tez a delegate has, both from its own account and from delegators, the more likely it is to be selected to bake a block and thus receive the rewards.
+A delegate participates in consensus in proportion to their _baking power_: the more baking power a delegate has, the more likely it is to be selected to bake or to validate blocks and thus receive the rewards.
+The baking power of a delegate is computed from the amounts of tez owned by its own and by all the accounts that delegate to it, knowing that non-staked tez are weighted half as much as staked tez in the sum.
+
+The delegate must have a baking power of at least 6,000 tez to be allowed to bake.
+
+A delegate also participates in [governance](./governance) in proportion to their _voting power_.
+The voting power of a delegate is similar to the baking power except that no distinctions is made between staked and non-staked tez.
 
 Bakers must run at least one Tezos node and a baker daemon to go with it.
-These daemons must run at all times with a stable internet connection, because inactive bakers are automatically removed from the network.
+These daemons must run at all times with a stable power source and internet connection, because periods of inactivity for a baker entail losses of rewards, and eventually being marked as inactive and temporarily excluded from baking.
 
 ## Delegating to a baker
 
-If you don't have enough tez to become a baker or don't want to run a baking node, you can delegate tez to a baker, which makes you a _delegator_.
-The delegate doesn't have control over your tez and you can spend your tez at any time or withdraw your delegation, but the tez that you delegate counts toward the amount of tez that the delegate has for baking purposes.
-Delegators receive a share of the delegate's rewards in proportion to the amount of tez that they delegate.
+If you don't have enough tez to become a baker or don't want to run a baking node, you can choose a baker as your delegate, which makes you a _delegator_.
+The delegate doesn't have control over your tez and you can spend your tez at any time or withdraw your delegation, but half of the tez that you delegate counts toward the baking power of the delegate. Also, delegated tez increase the voting power of your baker: a delegate's voting power is the sum of its own tez plus the tez delegated to it.
+
+In exchange, delegates may share some part of their rewards with you, in proportion to the amount of tez that you currently have.
+Check your delegate's conditions for distributing rewards.
+
+Delegating incurs no risk for the delegator: the delegate has no control over the tez and the delegator is not punished if the delegate misbehaves.
+
+## Staking with a baker
+
+If you want to get a bigger share of your delegate's rewards, you can temporarily stake any amount of tez from your account on behalf of your baker.
```

Comment:
```suggestion
If you want to get a bigger share of your delegate's rewards, you can temporarily stake any amount of tez from your account on behalf of your baker.
```
with your baker? We use this in other comms.

Reply:
changed to "with"